usaspending_api/data_load/fpds_loader.py

```diff
@@ -0,0 +1,260 @@
+    with psycopg2.connect(dsn=USASPENDING_CONNECTION_STRING) as connection:
+        with connection.cursor(cursor_factory=psycopg2.extras.DictCursor) as cursor:
+
+            # BLIND INSERTS
+            # First create the records that don't have a foreign key out to anything else in one transaction per type
+            inserted_recipient_locations = insert_recipient_locations(cursor, load_objects)
+            for index, elem in enumerate(inserted_recipient_locations):
+                load_objects[index]["legal_entity"]["location_id"] = inserted_recipient_locations[index]
+
+            inserted_recipients = insert_recipients(cursor, load_objects)
+            for index, elem in enumerate(inserted_recipients):
+                load_objects[index]["transaction_normalized"]["recipient_id"] = inserted_recipients[index]
+                load_objects[index]["award"]["recipient_id"] = inserted_recipients[index]
+
+            inserted_place_of_performance = insert_place_of_performance(cursor, load_objects)
+            for index, elem in enumerate(inserted_place_of_performance):
+                load_objects[index]["transaction_normalized"][
+                    "place_of_performance_id"
+                ] = inserted_place_of_performance[index]
+                load_objects[index]["award"]["place_of_performance_id"] = inserted_place_of_performance[index]
+
+            # Handle transaction-to-award relationship for each transaction to be loaded
+            for load_object in load_objects:
+
+                # AWARD GET OR CREATE
+                award_id = _lookup_award_by_transaction(cursor, load_object)
+                if not award_id:
+                    # If there is no award, we need to create one
+                    award_id = insert_award(cursor, load_object)
+
+                load_object["transaction_normalized"]["award_id"] = award_id
+                ids_of_awards_created_or_updated.add(award_id)
+
+                # TRANSACTION UPSERT
+                transaction_id = _lookup_existing_transaction(cursor, load_object)
+                if transaction_id:
+                    # Inject the Primary Key of transaction_normalized+transaction_fpds that was found, so that the
+                    # following updates can find it to update
+                    load_object["transaction_fpds"]["transaction_id"] = transaction_id
+                    _update_fpds_transaction(cursor, load_object, transaction_id)
+                else:
+                    # If there is no transaction we create a new one.
+                    transaction_id = _insert_fpds_transaction(cursor, load_object)
+
+                load_object["transaction_fpds"]["transaction_id"] = transaction_id
+                load_object["award"]["latest_transaction_id"] = transaction_id
+
+    return list(ids_of_awards_created_or_updated)
+
+
+def _lookup_award_by_transaction(cursor, load_object):
+    # Try to find an award for this transaction to belong to
```

Comment:
Could some of these comments (ln 224, 233, 244-245, 252) be changed to docstrings? (some are already docstring-y, others would need to be re-written) Not a big deal either way.